markdown/preface.markdown

```diff
@@ -363,27 +378,27 @@ In essence, GitHub has democratized software development. It has substantially r
 
 ### Make your Mark on the SciPy Ecosystem
 
-Once you have become more comfortable with SciPy and started using it for your research, you may find that a particular package is lacking a feature you need, or you think that you can do something more efficiently, or perhaps you've found a bug.
+As you gain more experience with SciPy and start using it for your research, you may find that a particular package is lacking a feature you need, or you think that you can do something more efficiently, or perhaps find a bug.
 When you reach this point, it's time to start contributing to the SciPy Ecosystem.
 
 We strongly encourage you to try doing this.
 The community lives because people are willing to share their code and improve existing code.
-Beyond any altruistic reasons for contributing, there are some very practical personal benefits.
+But, beyond any altruistic reasons for contributing, there are some very practical personal benefits.
 By engaging with the community you will become a better coder.
 Any code you contribute will be reviewed by others and you will receive feedback.
 As a side effect, you will learn how to use Git and GitHub, which are very useful tools for maintaining and sharing your own code.
 You may even find that interacting with the SciPy community provides you with a broader scientific network, and surprising career opportunities.
 
-Later in Elegant SciPy we will show you how to contribute your new skills to the GitHub-hosted projects that comprise most of the scientific Python ecosystem.
-In the meantime we want you to start thinking about being more than just a SciPy user.
-You are joining a community and we hope you will keep making it a better place for all scientific coders.
+Later in Elegant SciPy we will show you how to use your new skills to contribute to the GitHub-hosted projects that comprise most of the scientific Python ecosystem.
+In the meantime, we want you to think about being more than just a SciPy user.
+You are joining a community, and we hope you will make it a better place for all scientific coders.
```

Comment:
How do you feel about, "and your work will make it a better place for all scientific coders."